src/HttpRequestHandler.cpp

```diff
@@ -1,11 +1,17 @@
 #include "HttpRequestHandler.hpp"
+#include "FileHandler.hpp"
```

Comment:
btw, should we start putting these includes into the respective hpp? like put this include to the #include for Filehandler to the HttpRequestHandler.hp

Reply:
No need to make the change though, lets fo it later